x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/planner/QueryTranslator.java

```diff
@@ -459,10 +459,10 @@ protected QueryTranslation asQuery(MultiMatchQueryPredicate q, boolean onAggs) {
         }
     }
 
-    static class BinaryLogic extends ExpressionTranslator<BinaryPredicate> {
+    static class BinaryLogic extends ExpressionTranslator<org.elasticsearch.xpack.sql.expression.predicate.logical.BinaryLogic> {
```

Comment:
Maybe I'm missing something but why do you need fully qName here?

Reply:
Name clash on `BinaryLogic`.